pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: ../common/yamllint
+platforms:
+  - name: instance
+    image: solita/ubuntu-systemd:latest
+    command: /sbin/init
+    capabilities:
+      - SYS_ADMIN
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
```

Comment:
`privileged` is superset of `SYS_ADMIN` capability so one of them could be dropped. Also it seems like `SYS_ADMIN` is enough, so it's better to prefer `privileged`: https://molecule.readthedocs.io/en/latest/examples.html#systemd-container

Reply:
ok